requirements-opt.txt

```diff
@@ -39,3 +39,7 @@ psycopg2==2.6
 # It's a pure python module, it doesn't require anything to install,
 # but needs the pywin32 extension to work
 wmi==1.4.9
+
+# checks.d/snmp/pu
+# Require gevent
+gevent==1.0.1
```

Comment:
Don't forget to reflect this add-on on [dd-agent-omnibus](https://github.com/DataDog/dd-agent-omnibus/blob/d0ade317b5956ef17c6ead007655a0c65cdafa11/config/projects/datadog-agent.rb#L136-L154) and [omnibus-software](https://github.com/DataDog/omnibus-software/tree/master/config/software) :wink: .